tests/integration-tests/tests/schedulers/test_slurm_accounting.py

```diff
@@ -173,11 +173,24 @@ def test_slurm_accounting(
     config_params = _get_slurm_database_config_parameters(database_stack_outputs)
     public_subnet_id = vpc_stack_for_database.get_public_subnet()
     private_subnet_id = vpc_stack_for_database.get_private_subnet()
-    cluster_config = pcluster_config_reader(
-        public_subnet_id=public_subnet_id, private_subnet_id=private_subnet_id, **config_params
-    )
+
+    # First create a cluster without Slurm Accounting disabled
+    cluster_config = pcluster_config_reader(public_subnet_id=public_subnet_id, private_subnet_id=private_subnet_id)
     cluster = clusters_factory(cluster_config)
 
+    remote_command_executor = RemoteCommandExecutor(cluster)
+
+    _test_that_slurmdbd_is_not_running(remote_command_executor)
+
+    # Then update the cluster to enable Slurm Accounting
+    updated_config_file = pcluster_config_reader(
+        config_file="pcluster.config.update.yaml",
+        public_subnet_id=public_subnet_id,
+        private_subnet_id=private_subnet_id,
+        **config_params,
+    )
+    cluster.update(str(updated_config_file), force_update="true")
```

Comment:
why do we need a forced update? Is it because the update is not officially supported? If this is the case, I suggest to add a comment specifying it and create a task to investigate what are the blockers to officially support it.

Reply:
The force_update was used in the original test so I carried it over, but I am testing to see if it works without.